src/sentry/utils/samples.py

```diff
@@ -182,7 +184,7 @@ def load_data(platform, default=None, timestamp=None, sample_name=None):
 
     # Make breadcrumb timestamps relative to right now so they make sense
     breadcrumbs = data.get('sentry.interfaces.Breadcrumbs')
-    if breadcrumbs is not None:
+    if breadcrumbs is not None and 'values' in breadcrumbs:
```

Comment:
It this a change that creeped in or does this PR change behavior WRT to values?

Reply:
That was a defensive change to keep behavior of the cocoa ingestion test the same. `cocoa.json` contains a key `breadcrumbs: [ ... ]`, which was not touched by the old code and would error with the new one. So instead of fixing this code, I chose to rather keep the test behavior the same.

Reply:
Is the cocoa dump the only one that does not have canonical data for value use? If that's the case I rather have the data fixed than special handle this here.

Reply:
Sure thing. Will change the visual diff though.